packages/forma-36-react-components/src/components/Tooltip/Tooltip.tsx

```diff
@@ -176,6 +187,34 @@ export const Tooltip = ({
 
   // const delay = !hideDelay ? (closeOnMouseLeave ? 0 : 300) : hideDelay;
```

Comment:
I know you didn't introduce this here, but could you remove it in this PR? 🙏

Reply:
Oops, auto merge 😓 I will remove it.